backend/src/main/java/com/daedan/festabook/place/domain/Place.java

```diff
@@ -13,15 +14,26 @@
 import jakarta.persistence.Id;
 import jakarta.persistence.JoinColumn;
 import jakarta.persistence.ManyToOne;
+import java.time.LocalTime;
+import java.util.Set;
 import lombok.AccessLevel;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import org.springframework.http.HttpStatus;
 
 @Entity
 @Getter
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
 public class Place {
 
+    private static final Set<PlaceCategory> MAIN_PLACE =
+            Set.of(PlaceCategory.BAR, PlaceCategory.BOOTH, PlaceCategory.FOOD_TRUCK);
```

Comment:
줄바꿈 말고 다음 방법도 있을 것 같아요
```suggestion
    private static final Set<PlaceCategory> MAIN_PLACE = Set.of(
        PlaceCategory.BAR, 
        PlaceCategory.BOOTH, 
        PlaceCategory.FOOD_TRUCK
    );
```

Reply:
좋아요~~